pexv2/pd.go

```diff
@@ -0,0 +1,46 @@
+package pexv2
+
+// PresentationDefinition represents a DIF Presentation Definition defined [here].
+// Presentation Definitions are objects that articulate what proofs a Verifier requires
+//
+// [here]: https://identity.foundation/presentation-exchange/#presentation-definition
+type PresentationDefinition struct {
+	ID               string            `json:"id,omitempty"`
```

Comment:
```suggestion
	ID               string            `json:"id"`
```

not that we really get much from this, as it doesn't enforce requirement of the properties, but for the sake of consistency

Reply:
Good catch @KendallWeihe !